### PR TITLE
Added ORM mapping and security checks in the CI and Makefile

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -56,11 +56,14 @@ test:
             parallel: true
         - ./bin/console doctrine:schema:drop --force --env=test_mysql:
             parallel: true
-        - ./bin/console doctrine:schema:create --env=test_mysql:
+        - ./bin/console doctrine:migrations:migrate -n --env=test_mysql:
```
Comment (B-Galati) [inline circle.yml]: Could you also update the Makefile to use migrations in test (task `tfp`) instead of schema:create ?

Comment (HeahDude) [reply circle.yml]: Done.

File: circle.yml [hunk 2]
```diff
@@ -72,6 +75,9 @@ test:
         # Try to build production JS
         - yarn run build-prod
 
+        # Check for vulnerable dependencies
+        - ./vendor/bin/security-checker security:check
```
Comment (Remg) [inline circle.yml]: 👍 
Should we remove the same check in the `build.sh` script then ?
https://github.com/EnMarche/en-marche.fr/blob/master/scripts/build.sh#L5
I don't see the point of checking it during the build 🤔

Comment (HeahDude) [reply circle.yml]: Done.

**Comment by thomasbisignani** [comment]
Nice :ok_hand:

**Pull request merged**